source/administration/config-settings.rst

```diff
@@ -1183,20 +1183,150 @@ Advanced Logging
+|               |                                                                                                                                                        |             |
+| IP            | IP address or domain of the socket server.                                                                                                             | string      |
++---------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+-------------+
+|               |                                                                                                                                                        |             |
+| Port          | Listening port of the socket server.                                                                                                                   | int         |
++---------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+-------------+
+|               |                                                                                                                                                        |             |
+| TLS           | Set to ``true`` to connect via TLS. Set to ``false`` to prevent connecting via TLS.                                                                    | bool        |
++---------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+-------------+
+|               |                                                                                                                                                        |             |
+| Cert          | For TLS connections where TLS is set to ``true``, the filename of client certificate or base64-encoded certificate.                                    | string      |
++---------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+-------------+
+|               |                                                                                                                                                        |             |
+| Insecure      | Used for testing purposes only. Set to ``true`` to prevent a certificate check from being performed. Set to ``false`` to perform a certificate check.  | bool        |
++---------------+--------------------------------------------------------------------------------------------------------------------------------------------------------+-------------+
+
+.. Note::
+    Filenames for ``AdvancedLoggingConfig`` can be absolute or relative, and can also contain the JSON embedded directly.
```

Comment:
I would word this something like, "AdvancedLoggingConfig can contain a absolute or relative filename, or embedded JSON."